### ISSUE TITLE
[Standalone] Some comboboxes have wrong colors

### ISSUE THREAD

**Issue by jonjamcam** [elieserdejesus/JamTaba#952]
Flat theme:

- [x] .
 ![image](https://user-images.githubusercontent.com/15310433/31894613-5859502c-b7e5-11e7-9cdc-ff6f6e7b02f4.png)

- [x] .
 ![image](https://user-images.githubusercontent.com/15310433/31898431-895b8fc8-b7ef-11e7-8bf8-0a3462cbe392.png)

Ice theme:

- [x] .
 ![image](https://user-images.githubusercontent.com/15310433/31894650-6b068fbe-b7e5-11e7-937c-0f8d2966a9c1.png)

EDIT: Sometimes if you use flat or rounded theme from start, you get the combobox options cropped (looper in this case):

![image](https://user-images.githubusercontent.com/15310433/31895708-4d55ac36-b7e8-11e7-9315-e4f1a859c75a.png)

(couldn't reproduce this last one again)

FIxing in https://github.com/elieserdejesus/JamTaba/pull/953

**Comment by elieserdejesus** [comment]
@jonjamcam , I never see these black combos here.  Maybe the CSS are mixed in your machine?  Or in my machine? 

Can you try a very clean compilation? Manually delete the Themes folder from appData and try qmake, clean, compile to see if you sill have the problem.

**Comment by jonjamcam** [comment]
Yes. To be sure I deleted the themes folder and did a clean compilation. I see other changes too in comboboxes menu since navy theme was merged, but I'm not sure if this is the cause. For example now I'm unable to modify colors inside the dropdown (abstractview) menu of the comboboxes and the font is a little different than before.

**Comment by jonjamcam** [comment]
@elieserdejesus I compiled the vst plugin and no problems there. In my machine this issue happens only on standalone

**Comment by elieserdejesus** [comment]
@jonjamcam , I confirmed the issue here. I deleted the Theme folder and I can see black comboboxes in Flat theme, preferences panel.

**Comment by elieserdejesus** [comment]
This line is the problem:
https://github.com/elieserdejesus/JamTaba/blob/master/src/Standalone/main.cpp#L31

I setted the Qt style to "fusion" to solve some stylesheet differences between Windows and Mac. Using the "fusion" style Qt applications will have the same visual in all plataforms. 

But looks like this affected some widgets.

@jonjamcam , can you fix these combos stylesheets?

**Comment by jonjamcam** [comment]
> can you fix these combos stylesheets?

yes. But I'm afraid the fixes will have an effect on the vst plugin that is spot on. I will try to keep it the least intrusive possible. Ohh and BTW some code will not work like `QComboBox QAbstractItemView`

**Comment by elieserdejesus** [comment]
@jonjamcam , I see some weird behaviors in QComboBox stylesheet when using alpha (rgba). I have some results with this code in **General.css**. I avoided almost all **alpha colors** and moved the **selection-** properties from QAbstractItemView to QComboBox:
```
QComboBox
{
    color: rgb(200, 200, 200);
    border-radius: 6px;
    border: 1px solid #33779E;
    background-color: #3F92C1;

    selection-color: black;
    selection-background-color: #3F92C1;
}

QComboBox QAbstractItemView
{
    border: 1px solid transparent;
    background-color: rgba(0, 0, 0, 30);
}
```
![image](https://user-images.githubusercontent.com/1012741/31919473-7398d662-b841-11e7-8953-00c20bd3402a.png)

**Comment by jonjamcam** [comment]
Great! I'll try that

**Comment by elieserdejesus** [comment]
closed in Fixing wrong comboboxes colors #953
